django-stubs/dispatch/dispatcher.pyi

```diff
@@ -1,16 +1,22 @@
+import threading
 from collections.abc import Callable
+from logging import Logger
 from typing import Any
+from weakref import WeakKeyDictionary
 
-NONE_ID: Any
+NONE_ID: int
 NO_RECEIVERS: Any
 
+logger: Logger
+
 class Signal:
-    receivers: Any
-    providing_args: Any
-    lock: Any
-    use_caching: Any
-    sender_receivers_cache: Any
-    def __init__(self, providing_args: list[str] = ..., use_caching: bool = ...) -> None: ...
+    receivers: list[Any]
+    lock: threading.Lock
+    use_caching: bool
+    sender_receivers_cache: WeakKeyDictionary[Any, Any] | dict[Any, Any]
+    _dead_receivers: bool
```

Comment:
Usually we don't annotate protected attributes, any reason to make an exception?

Reply:
No specific reason, but it's type was easily inferable. I'll remove it for consistency.